src/components/select/simple-select/simple-select.spec.js

```diff
@@ -646,6 +647,16 @@ describe('SimpleSelect', () => {
       });
     });
   });
+
+  it('the required prop is passed to the input', () => {
+    const wrapper = renderSelect({ label: 'required', required: true });
+
+    const input = wrapper.find('input');
+    expect(input.prop('required')).toBe(true);
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.